utils/utils_test.go

```diff
@@ -0,0 +1,12 @@
+package utils
+
+import (
+	"testing"
+
+	"_vendor/github.com/stretchr/testify/assert"
```

Comment:
I think this import is also an issue. This import implies a GOPATH that's set to gillnet/go, and that uses our old _vendor model.